src/4_replace_array.js

```diff
@@ -1,11 +1,17 @@
-module.exports = function replaceArray() {
-  // TODO Implement me.
+module.exports = replaceArray
+
+function replaceArray(inputTab) {
+  let outputTab = []
+  for(el of inputTab) {
+      el[0].includes("h") || el[0].includes("H") ? outputTab.push(el.toUpperCase()) : outputTab.push(el)
+  }
```

Comment:
IMO trenary w tej sytuacji powinno być użyte jak niżej (zazwyczaj raczej chcemy zwrócić element niż wywoływać funkcję w trenary):

```suggestion
  for(el of inputTab) {
      const e = el[0].includes("h") || el[0].includes("H") ? el.toUpperCase() : el;
      outputTab.push(e)
  }
```

Reply:
ok